docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
+
+3.      Design
+Here the software architecture is defined, describing the high-level structure, components, interfaces, and interactions of the software system. The architectural design lays the groundwork for the detailed design and implementation phases.
+
+4.      Development
+The development stage will be the big bulk of a project, here code is created that will eventually define the software.
+
+  a.       Usage of a decentralized code management system, such as git, together with a centralised repository, such as Github, Gitlab or Bitbucket, that is seamlessly integrated with a task management system, is strongly recommended.
+
+  b.       Have a coding policy that covers which language is used, how the language is formatted, do’s and don’ts, the use of inline documentation. This policy is not to be strictly enforced, but will open discussions on how the team likes its code and will lead to a more concise and unified coding strategy.
+
+  c.       Write code with unit and integration tests. Moreover, conducting regular code reviews is highly recommended. These steps are crucial for code verification and will be invaluable during software maintenance, testing, and release phases. It enables a wider understanding of parts of the code as unit tests often embody general usage as well as very specific exceptions and code reviews enforce colleagues to have a look and understand what your code is doing.
+
+  d.       Establish a well-defined development and release cycle with set time periods for preparation, development, testing & releasing. This allows multiple developers to work cohesively on the project and creates structured testing and release moments during the implementation and maintenance processes. It also enables ample opportunities for user reflections on the deliverables during the lifetime of project.
+
+5.      Testing
+Before implementation, software should be extensively tested, verified and validated. While ISO 15189 does not provide specific guidelines on how to conduct software validation, it emphasizes the importance of ensuring that software is validated and verified following recognized industry standards. We advise the use of the software validation and verification procedures as described in this guideline.
+
+6.      Implementation
+Once everything is ready, a clear time point should be determined to release the software. The exact time point at which a software is released can be determined by the Development & Release cycle or released in an independent matter. The software functionality and release date should be clearly communicated with the end users.
```

Comment:
General; we should mention what needs to be documented.
Or state somewhere that documentation needs to be present for all subsections in how this is done within the lab

Reply:
what needs to be documented with a release;  how to do this..
pull requests / merge requests / checkout location should be documented

Reply:
I have added an introduction that stated that laboratories should establish documented guidelines covering all SDLC phases. I have also emphasized that the process on how to perform a release should be well defined